cli/test/smokehouse/test-definitions/a11y.js

```diff
@@ -818,6 +866,10 @@ const expectations = {
           ],
         },
       },
+      'skip-link': {
+        score: null,
```

Comment:
Why couldn't this be tested? Is it because there is no `main`?

Reply:
I was thinking it could be useful to know that having no skip link means this audit is not applicable rather than failed (which isn't obvious from the [doc wording](https://dequeuniversity.com/rules/axe/4.7/skip-link)). But just updated this to have a test with a non-focusable skip link.